### DISCUSSION TITLE
Patrol with Firebase test lab

### DISCUSSION THREAD

**Discussion by jdpadillaac** [leancodepl/patrol#706] [Q&A] [answered]
Patrols test works with Firebase test lab? Like integration_test flutter package

Integration_test does not support native interaction, but I need to use android test instrumentation run in Firebase test lab or Device farm on AWS.

**Comment by bartekpacia** [comment, marked as answer]
Hi @jdpadillaac,

Yes, Patrol does work with Firebase Test Lab – [see the docs here](https://patrol.leancode.co/ci).

See also these issues:
- #332
- #622
